playbook-website/package.json

```diff
@@ -24,36 +34,35 @@
     "@tiptap/pm": "^2.0.2",
     "@tiptap/react": "^2.0.2",
     "@tiptap/starter-kit": "^2.0.2",
-    "anchor-js": "^5.0.0",
-    "maplibre-gl": "^2.4.0",
-    "playbook-ui": ">= 1",
+    "@vitejs/plugin-react": "^4.3.1",
+    "axe-core": "4.9.1",
+    "babel-plugin-flow-runtime": "^0.11.1",
+    "babel-plugin-react-docgen": "^4.2.1",
+    "babel-plugin-transform-flow-strip-types": "^6.22.0",
+    "babel-plugin-transform-react-remove-prop-types": "^0.4.24",
+    "babel-preset-flow": "6.23.0",
+    "copy-to-clipboard": "^3.3.3",
+    "eslint": "6.8.0",
+    "postcss-flexbugs-fixes": "^5.0.2",
+    "postcss-import": "^14.1.0",
+    "postcss-preset-env": "^7.0.1",
     "prismjs": "^1.29.0",
     "react": "^17.0.2",
     "react-docgen": "^7.0.3",
+    "react-docgen-typescript": "^2.2.2",
     "react-dom": "^17.0.2",
     "react-router-dom": "^6.16.0",
     "react-simple-code-editor": "^0.13.1",
+    "react-svg-loader": "^3.0.3",
+    "sass": "^1.77.6",
+    "style-loader": "2.0.0",
     "trix": "1.3.1",
-    "webpacker-react": "^0.3.2",
+    "ts-json-schema-generator": "^2.3.0",
+    "vite": "^5.0.0",
+    "vite-plugin-ruby": "^5.0.0",
```

Comment:
There were many deps here that were in the dependencies area which are actually just devDependencies.